_install_x86/etc/init.d/rcS

```diff
@@ -1,8 +1,11 @@
 mkdir -p /sys
 mkdir -p /tmp
 mkdir -p /proc
+mkdir -p /mnt
+mkdir -p /lib/modules/4.0.0+
 /bin/mount -a
 mkdir -p /dev/pts
 mount -t devpts devpts /dev/pts
 echo /sbin/mdev > /proc/sys/kernel/hotplug
 mdev -s
+mount -t 9p -o trans=virtio kmod_mount /mnt
```

Comment:
it had better do this mount by yourself , maybe some guys have not insmod those 9p kernel modules

Reply:
Alright! I consider that new comers don't know how to mount that. So I add the command to rcS. Even someone doesn't insmod the module for 9p, it doesn't influence the minimal rootfs to boot up. And also the defconfig is configured that to y as default value.